zk_toolbox/crates/common/src/prerequisites.rs

```diff
@@ -26,6 +26,10 @@ const PREREQUISITES: [Prerequisite; 6] = [
         name: "yarn",
         download_link: "https://yarnpkg.com/getting-started/install",
     },
+    Prerequisite {
+        name: "pg_isready",
+        download_link: "https://www.postgresql.org/download",
+    },
 ];
```

Comment:
let's implement a simple function like here
https://github.com/popzxc/zk_stack_wizard/blob/3c0a5810490b27072cb458a65746dca4120e210c/src/init.rs#L125-L138

instead of more deps